src/main/java/org/radarcns/util/commandline/CommandLineArgs.java

```diff
@@ -0,0 +1,28 @@
+package org.radarcns.util.commandline;
+
+import com.beust.jcommander.Parameter;
+
+public class CommandLineArgs {
+
+    @Parameter(names = { "-f", "--format" }, description = "Format to use when converting the files. JSON and CSV is available.")
+    public String format = "csv";
+
+    @Parameter(names = { "-c", "--compression" }, description = "Compression to use when converting the files. Gzip is available.")
+    public String compression = "none";
+
+    // Default set to false because causes loss of records from Biovotion data. https://github.com/RADAR-base/Restructure-HDFS-topic/issues/16
+    @Parameter(names = { "-d", "--deduplicate" }, description = "Boolean to define if to use deduplication or not.", validateWith = BooleanValidator.class)
+    public Boolean deduplicate = false;
+
+    @Parameter(names = { "-u", "--hdfs-uri" }, description = "The HDFS uri to connect to. Eg - 'hdfs://<HOST>:<RPC_PORT>/<PATH>'.", required = true, validateWith = { HdfsUriValidator.class, PathValidator.class })
+    public String hdfsUri;
+
+    @Parameter(names = { "-i", "--hdfs-root-directory" }, description = "The input HDFS root directory from which files are to be read. Eg - '/topicAndroidNew'", required = true, validateWith = PathValidator.class)
```

Comment:
How about we choose between having a URI or multiple paths? Having both a path and an URI seems overkill... So
```
command -u webhdfs://localhost -o output /topicAndroidNew/application_status /topicAndroidNew/application_timezone
command -u <uri> -o <dir> <input1>...
```